### ISSUE TITLE
Add a highlight around the recording window

### ISSUE THREAD

**Issue by Pawwwle** [akon47/ScreenRecorder#51]
Please add a highlight around the recording window to make the recording area visible. This is nicely solved, for example, in the FastStone Capture application.

**Comment by akon47** [comment]
Agreed — a visible highlight/border around the area being recorded would help a lot. On the wishlist for a future 2.x release (the 2.0 rewrite makes overlay features like this easier to add).
